lib/image-stream/utils.js

```diff
@@ -37,6 +38,10 @@ const archiveType = require('archive-type');
  */
 exports.getArchiveMimeType = (file) => {
 
+  if (mime.lookup(file)) {
```

Comment:
Depends on how much work it is. If leaving just `mime-types` makes the tests green, lets go for it here.